packages/ozone/src/api/moderation/emitEvent.ts

```diff
@@ -214,6 +220,32 @@ export default function (server: Server, ctx: AppContext) {
   })
 }
 
+const TAG_AUTH: Record<string, 'triage' | 'moderator' | 'admin'> = {
+  'chat-disabled': 'moderator',
+}
+
+const assertTagAuth = (
+  event: ModEventTag,
+  auth: ModeratorOutput | AdminTokenOutput,
+) => {
+  // admins can add/remove any tag
+  if (auth.credentials.isAdmin) return
+
+  for (const tag of Object.keys(TAG_AUTH)) {
+    if (event.add.includes(tag) || event.remove.includes(tag)) {
+      if (TAG_AUTH[tag] === 'admin' && !auth.credentials.isAdmin) {
+        throw new Error(`Must be an admin to add tag: ${tag}`)
+      } else if (
+        TAG_AUTH[tag] === 'moderator' &&
+        !auth.credentials.isModerator
+      ) {
+        throw new Error(`Must be a full moderator to add tag: ${tag}`)
+      }
+    }
+  }
+  return true
```

Comment:
No real issue, just think this can go.
```suggestion
```

Reply:
ah good looks - used to be an `is` function but converted it to `assert`